modules/importing-manifest-list-through-imagestreamimport.adoc

```diff
@@ -0,0 +1,69 @@
+// Module included in the following assemblies:
+// * assembly/openshift_images/managing-image-streams.adoc
+
+:_content-type: PROCEDURE
+[id="importing-manifest-list-through-imagestreamimport_{context}"]
+= Importing a manifest list through ImageStreamImport
+
+Use the following procedure to import a manifest list through the `ImageStreamImport` object with the `importMode` value. 
+
+.Procedure
+
+. Create an `ImageStreamImport` object that sets the `importMode` to `Manifestlist` by running the following command:
++
+[source,terminal]
+----
+$ cat <<EOF | oc create -f -
+apiVersion: image.openshift.io/v1
+kind: ImageStreamImport
+metadata:
+  name: app
+  namespace: <namespace>
+spec:
+  import: true
+  images:
+  - from:
+      kind: DockerImage
+      name: quay.io/<user_name>/<image_name>
```

Comment:
what do we usually have in docs when we need a container image? `quay.io/<user_name>/<image_name>` looks like only quay is supported here.

Reply:
@dmage sometimes we use 

name: inputimage:latest.

WDYT? we don't have a value that's broader than that, from what I can tell. This is my first time writing for ImageStreams, though